www/about_team.mako

```diff
@@ -1056,6 +1031,22 @@
             </ul>
         </div>
     </li>
+<li class="col-md-3 space-mobile">
+        <div class="thumbnail">
+            <img src="../static/img/people/davydova.jpg" alt="Katya Davydova">
+            <h3>
+                Katya Davydova
+                <small> Human Resources Intern | Community</small>
+                <noscript><span class="obfuscated-email-noscript"><small style="padding-top: 5px;">kdav<span style="display:none;">null</span>ydova@co<span style="display:none;">null</span>s.io</small></span></noscript>
```

Comment:
Sorry to be picky (should have been clearer about this one too) but can you remove this one too? It only shows up when a person has javascript disabled but still. Thanks!